### PR TITLE
Add simple SLSA identifier strings and provide integration with ingestor

### PR DESCRIPTION
**Pull request by lumjjb**
Implementing the parser side of the collectsub service where found string tokens that resemble potential metadata targets get registered in the collectsub service and the collectors are able to listen for. As part of #244 

Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

File: pkg/ingestor/parser/parser.go [hunk 1]
```diff
@@ -113,18 +113,30 @@ func Subscribe(ctx context.Context, transportFunc func([]assembler.Graph) error)
 }
 
 // ParseDocumentTree takes the DocumentTree and create graph inputs (nodes and edges) per document node.
-func ParseDocumentTree(ctx context.Context, docTree processor.DocumentTree) ([]assembler.Graph, error) {
+func ParseDocumentTree(ctx context.Context, docTree processor.DocumentTree) ([]assembler.Graph, []*common.IdentifierStrings, error) {
```
Comment (naveensrinivasan) [inline pkg/ingestor/parser/parser.go:116]: I firmly believe that having more than a tuple in return for a function makes it harder to maintain. I suggest we change this into a struct.

Comment (lumjjb) [reply pkg/ingestor/parser/parser.go:116]: Thanks for the review. I believe 2 values besides an error is reasonable. Will get input from comments from other maintainers as well.

**Review by mlieberman85 — APPROVED** [review]
LGTM

File: cmd/guacone/cmd/files.go [hunk 2]
```diff
@@ -212,10 +212,14 @@ func getProcessor(ctx context.Context) (func(*processor.Document) (processor.Doc
 }
 func getIngestor(ctx context.Context) (func(processor.DocumentTree) ([]assembler.Graph, error), error) {
 	return func(doc processor.DocumentTree) ([]assembler.Graph, error) {
-		inputs, err := parser.ParseDocumentTree(ctx, doc)
+		inputs, idStrings, err := parser.ParseDocumentTree(ctx, doc)
```
Comment (pxp928) [inline cmd/guacone/cmd/files.go:215]: is this needed? If not used can remove `idStrings` and move the below comment here.

**Review by pxp928 — APPROVED** [review]
LGTM. Small nits

**Pull request merged**